modules/apps/site-initializer/site-initializer-raylife-ap/site-initializer-raylife-ap-test/src/testFunctional/tests/RaylifeAPApplicationsStatus.testcase

```diff
@@ -48,6 +46,105 @@ definition {
 		}
 	}
 
+	@description = "Story ID: LPS-159688 - Verify if the details of the Summary on the Bound application view appears."
+	@priority = 4
+	test AssertCanOpenBoundAppDetails {
+		property test.name.skip.portal.instance = "RaylifeAPApplicationsStatus#AssertCanOpenBoundAppDetails";
+
+		// Skip portal instance due to LRQA-79129. Unskip after it is resolved
+
+		task ("Given the Applications page is open") {
+			RaylifeAPNavigator.gotoSidebarItem(itemName = "Applications");
+		}
+
+		if (IsElementPresent(key_product = "Auto", key_status = "Bound", locator1 = "RaylifeAP#APPLICATIONS_NUMBER_WITH_STATUS")) {
+			task ("When the search term is present") {
+				RaylifeAP.search(searchTerm = "AP");
+
+				RaylifeAP.clickInApplicationNumber(
+					product = "Auto",
+					status = "Bound");
+			}
+
+			task ("Then assert the Detail view page and go back to 'Test Test' User") {
+				AssertElementPresent(
+					key_tabTitle = "App Details",
+					locator1 = "RaylifeAP#TAB_TITLE");
+
+				Navigator.openSpecificURL(url = "http://localhost:8080/home");
+
+				User.logoutPG();
+
+				User.firstLoginPG();
+			}
+		}
+		else {
+			task ("Given the new Application flow is open") {
+				RaylifeApplicationModal.openNewApplicationFlow(
+					button = "Next",
+					insurance = "Personal");
+			}
+
+			task ("When the user fulfill all the creation forms and generate a quote") {
+				RaylifeApplicationModal.typeAllNewApplicationContactInfoFields(
+					apt = 0000,
+					city = "New York",
+					emailAddress = "test@liferay.com",
+					firstName = "Testing",
+					lastName = "Model",
+					phoneNumber = 0123456789,
+					state = "NY",
+					streetAddress = "Address",
+					zipCode = 12345);
+
+				RaylifeApplicationModal.typeAllNewApplicationVehicleInfoFields(
+					annualMileage = 50000,
+					make = "Volvo",
+					model = "v90 cross country",
+					ownership = "Own",
+					primaryUse = "Business",
+					year = 2019);
+
+				RaylifeApplicationModal.typeAllNewApplicationDriverInfoFields(
+					ageFirstLicenced = 18,
+					firstName = "Testing",
+					gender = "Female",
+					highestEducation = "Masters",
+					lastName = "Model",
+					maritalStatus = "Single",
+					millitaryAffiliation = "None",
+					occupation = "Engineer",
+					relationToContact = "Spouse");
+
+				RaylifeApplicationModal.typeAllNewApplicationCoverageInfoFields(
+					bodilyInjury = "$30,000 / $60,000",
+					collisionAmount = "$2,000",
+					comprehensiveAmount = "$750",
+					medicalAmount = "$1,000",
+					propertyDamage = "$25,000",
+					uninsuredOrUnderinsuredMBI = "$30,000 / $60,000",
+					uninsuredOrUnderinsuredMPD = "$150,000");
+
+				Button.click(button = "Generate Quote");
+			}
+
+			task ("And search for the new application") {
+				RaylifeAP.search(searchTerm = "Testing");
+
+				RaylifeAP.clickInApplicationNumber(
```

Comment:
Just a suggestion here: can you do this macro without using index? When we use index, we make our macro "weaker".

Reply:
Done